conode/README.md

```diff
@@ -1,56 +1,612 @@
-Navigation: [DEDIS](https://github.com/dedis/doc/tree/master/README.md) ::
-[Cothority](../README.md) ::
-Conode
+<!-- START doctoc.sh generated TOC please keep comment here to allow auto update -->
+<!-- DO NOT EDIT THIS SECTION, INSTEAD RE-RUN doctoc.sh TO UPDATE -->
+**:book: Table of Contents**
 
-# Conode
+- [Introduction](#introduction)
+  - [Server requirements](#server-requirements)
+  - [Network communication ](#network-communication-)
+- [Running a conode](#running-a-conode)
+  - [Environment setup](#environment-setup)
+  - [Configuration setup](#configuration-setup)
+    - [Option 1: :computer: Configuration setup with the command line](#option-1-computer-configuration-setup-with-the-command-line)
+    - [Option 2: :whale: Configuration setup with docker](#option-2-whale-configuration-setup-with-docker)
+    - [Post setup instructions](#post-setup-instructions)
+  - [Run your conode](#run-your-conode)
+    - [Option 1: :computer: Run with the command line](#option-1-computer-run-with-the-command-line)
+    - [Option 2: :whale: Run with docker](#option-2-whale-run-with-docker)
+    - [Option 3: `run_nodes.sh`](#option-3-run_nodessh)
+- [Operating a conode](#operating-a-conode)
```

Comment:
"Operating a conode" is very similar to "Running a conode". Maybe change it to "Maintaining a conode"?